src/python/pants_release/generate_github_workflows.py

```diff
@@ -33,6 +33,7 @@
 
 
 class Platform(Enum):
+    # NB: If a new addition is made to this list, add it to `src/python/pants/backend/plugin_development/pants_requirements.py`
```

Comment:
This file doesn't seem to contain any platform references at the moment, is this comment potentially out of date?